### ISSUE TITLE
v2.0.1 Adds extra slash to urls with a capital "A" followed by a "/" in them

### ISSUE THREAD

**Issue by ofagbemi** [jfromaniello/url-join#25]
Urls with a capital "A" followed by a slash are getting an extra slash appended. This doesn't appear to be the case with other capital letters or lower case "a".

This doesn't appear to be an issue in 2.0.0.

```javascript
const join = require('url-join')

// correct
join('http://example.com', 'A')
// http://example.com/A

// correct
join('http://example.com', 'A/')
// http://example.com/A/, correct

join('http://example.com/', 'A/', '/')
// http://example.com/A//

join('http://example.com', 'A', '/')
// http://example.com/A//

join('http://example.comA/', 'hello')
// http://example.comA//hello

join('http://example.comA', '/hello')
// http://example.comA//hello
```

**Comment by koalazak** [comment]
Same problem here but with a `3` in the end:
```
urljoin('http://192.168.10.108:3003', '/foo/bar');
```

return `http://192.168.10.108:3003//foo/bar` ¬¬

I think its related with this recent commit:
https://github.com/jfromaniello/url-join/commit/711add4e8af8fc97390adef14b9a4722cac5e70a

```%3A``` :p

**Comment by xtuc** [comment]
PR is there https://github.com/jfromaniello/url-join/pull/26

**Comment by AlexZeitler** [comment]
Just ran into this bug - when will this PR be accepted? Thanks @xtuc for fixing it!

**Comment by jfromaniello** [comment]
I just merged #26 and published as `v2.0.2`. Thanks everyone for reporting, confirming and fixing! 

I am deeply sorry I took so much time to merge and release.

**Comment by AlexZeitler** [comment]
Thanks! 👍
